assets/app/styles/_core.less

```diff
@@ -56,16 +56,9 @@ html, body {
 
 .console-os .navbar-project {
   border-bottom: 1px solid @navbar-os-project-border-color; 
-  padding-bottom: 5px;
+  padding-bottom: 20px;
   position: relative;
   background: @navbar-os-project-bg;
-  min-height: 85px;
```

Comment:
the min-height was in place to prevent layout jumping before the `.list("projects"...` completed... was it breaking something or did it just not seem necessary? To see the behavior, comment out the `DataService.list("projects"` and view a page without the label filter on it